client/my-sites/theme/main.jsx

```diff
@@ -548,10 +548,10 @@ class ThemeSheet extends React.Component {
 
 	renderSheet = () => {
 		const section = this.validateSection( this.props.section );
-		const { siteId, retired } = this.props;
+		const { id, siteId, retired } = this.props;
 
-		const analyticsPath = `/theme/:slug${ section ? '/' + section : '' }${
-			siteId ? '/:site_id' : ''
+		const analyticsPath = `/theme/${ id }${ section ? '/' + section : '' }${
```

Comment:
Is `id` a string or a number?

Reply:
it's a string, passed always as string and defined in `PropTypes` as string. And always corresponds to the theme slug.
I think it's used like this because the `/themes` API endpoint returns it as `id` and not as `slug`. Will change it on a janitorial PR.

Reply:
one example of how it's used:

https://github.com/Automattic/wp-calypso/blob/a34f9c98086001052016f664eccd3a6d7d76a0dc/client/my-sites/theme/main.jsx#L164